### ISSUE TITLE
Type-o in ios/ReactNativeMatomo.swift

### ISSUE THREAD

**Issue by jeffphipps** [mccsoft/react-native-matomo#8]
There is a type-o in the current release in ios/ReactNativeMatomo.swift.  It has witchCachedQueue instead of withCachedQueue

**Comment by lorenc-tomasz** [comment]
Created PR that fixes it: https://github.com/mccsoft/react-native-matomo/pull/9 At the moment you can use `patch-package` to fix it.

**Comment by lorenc-tomasz** [comment]
Patch file:

```
diff --git a/node_modules/@mccsoft/react-native-matomo/ios/ReactNativeMatomo.swift b/node_modules/@mccsoft/react-native-matomo/ios/ReactNativeMatomo.swift
index fed3fe6..7688d7a 100644
--- a/node_modules/@mccsoft/react-native-matomo/ios/ReactNativeMatomo.swift
+++ b/node_modules/@mccsoft/react-native-matomo/ios/ReactNativeMatomo.swift
@@ -6,7 +6,7 @@ class ReactNativeMatomo: NSObject {
 
     var tracker: MatomoTracker!
 
-    @objc(initialize:withId:witchCachedQueue:withResolver:withRejecter:)
+    @objc(initialize:withId:withCachedQueue:withResolver:withRejecter:)
     func initialize(
         url:String,
         id:NSNumber,

```

**Comment by mav10** [comment]
Fixed by in version 1.0.6 #9
